lua/weapons/hands.lua

```diff
@@ -0,0 +1,52 @@
+SWEP.PrintName = "Hands"
+SWEP.Author = "TAS Team"
+SWEP.Contact = "https://taservers.com/"
+SWEP.Instructions =
+	"Right-click while walking and crouching to toggle the crosshair"
+SWEP.IconOverride = "entities/weapon_fists.png"
+SWEP.Spawnable = true
+SWEP.AdminSpawnable = false
+
+SWEP.Slot = 1
+SWEP.SlotPos = 1
+SWEP.HoldType = "normal"
+SWEP.ViewModel = "models/weapons/c_arms.mdl"
+SWEP.Primary.Ammo = "none"
+SWEP.Secondary.Ammo = "none"
+
+if CLIENT then
+	SWEP.DrawAmmo = false
+	SWEP.DrawCrosshair = true
+
+	function SWEP:DrawWorldModel() end
+	function SWEP:DrawWorldModelTranslucent() end
+else
+	function SWEP:ShouldDropOnDie()
+		return false
+	end
+
+	function SWEP:OnDrop()
+		if SERVER then
+			self:Remove()
+		end
```

Comment:
Ooh actually I don't think we need this `if SERVER` check now that the hook is only getting added serverside?

Reply:
Oh yeah, I missed that. Sorry.